proto/iotics/api/meta.proto

```diff
@@ -29,6 +29,12 @@ service MetaAPI {
   //   choose to end the stream early once they have received enough results. (E.g. in the case of Scope.LOCAL this
   //   would be after the one and only sequence of chunks has been received.)
   rpc SparqlQuery(SparqlQueryRequest) returns (stream SparqlQueryResponse) {}
+
+  // SparqlUpdate performs a SPARQL 1.1 query with write access to a local custom metadata graph and returns a result. 
```

Comment:
1. a SPARQL 1.1 _update_
2. I think _and returns a result_ might be redundant since there is nothing to return (other than error on failure).